man/setDT.Rd

```diff
@@ -58,6 +59,21 @@ setDT(X, key="a")[]
 X = list(a=1:5, a=6:10)
 setDT(X, check.names=TRUE)[]
 
+# FAQ: Dealing with data.table loaded from RDS or RData files
+# Example to demonstrate the usage of setDT after loading from RDS
+# Step 1: Define file path
+rds_file <- "my_data.rds"
```

Comment:
[Let's not touch files in the current directory.](https://contributor.r-project.org/cran-cookbook/code_issues.html#writing-files-and-directories-to-the-home-filespace) It's good that you've taken steps to avoid overwriting an important file belonging to the user, but CRAN still counts this as "leaving detritus". Instead, let's place the file in the session temporary directory:

```suggestion
rds_file = tempfile("my_data.rds_")
```

...and clean it up afterwards:

```r
unlink(rds_file); rm(rds_file)
```

Same goes for `truelength.Rd`.